packages/export/src/excel.js

```diff
@@ -8,6 +8,15 @@ let headerBackgroundColor = '#999999'
 let indexColumnBackgroundColor = '#bbbbbb'
 
 const convertToExcelCell = (value, index) => {
+  if (value?.meta?.__isHyperlink) {
+    const excelSafeUrl = value?.url?.replace(/"/g, '""') || ''
```

Comment:
##Blocker 

URLs should never contain " since it’s invalid in a URL. For separation of concerns, this code should assume a valid, well‑formed URL is provided by the call site rather than handling quote escaping here.